### DISCUSSION TITLE
so-status -q

### DISCUSSION THREAD

**Discussion by ericchiles** [Security-Onion-Solutions/securityonion#7952] [Unsupported Versions] [answered]
Running _so-status -q_ doesn't return anything. The help for so-status says it should return a single status code for the overall status of SO. However, when running this command, there is no status returned at all. Does the output go somewhere other than printing to the terminal, or is this functionality not working as intended?

**Comment by dougburks** [comment, marked as answer]
You can check the status code using `echo $?`:
```
sudo so-status -h

    /usr/sbin/so-status  [-h] [-q|--quiet]

   -h             Show this message.
   -q|--quiet     Suppress the output and only return a
	              single status code for overall status
	0:Ok, 1:Error, 2:Starting/Pending, 99:Installing SO

sudo so-status -q
echo $?
0
```
